package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Cell Tower Coverage",
```

Comment:
When I tried to run `npm install` I got this message: `npm WARN Invalid name: "Cell Tower Coverage"`. Sorry if I steered you wrong on the naming convention. I'm guessing that you need to remove the spaces.